lib/elixir/src/elixir_utils.erl

```diff
@@ -33,6 +33,14 @@ get_line(Opts) when is_list(Opts) ->
     false -> 0
   end.
 
+get_generated(Opts) when is_list(Opts) ->
+  case erlang:system_info(otp_release) of
+    Release when Release >= "19" ->
```

Comment:
Are we sure this is something we want to do now? AFAIK generally code is always compiled the same regardless of the release of OTP you built with, and this changes that. Perhaps this is better saved until whenever the plan is to drop support for 18?